docs/usersGuide/walkthrough.rst

```diff
@@ -71,12 +71,10 @@ The ini files are explained in detail in :doc:`../config/applicationProperties`.
 
 .. code-block:: ini
 
-    [COMMON]
```

Comment:
Why did you remove this? I think it should still be explained.

Reply:
1. Maybe this option is not so important and can be explained in the applicationProperties.rst.
2. It was not explained here anyway.

If this is supposed to be a kind of minimal walkthrough, this option/section may also be dropped. What do you think?

Reply:
Ah ok, I see it. It's ok then.